### PR TITLE
#77 and #80

### PR DESCRIPTION
**Pull request by IsmaelMartinez**
Merging the changes to add the ability to start teams with a set of pre-provided customCSS files for #77 .

Fixing the issue for #80 to add the ability to have a `config.json` file

### CODE REVIEW

**Comment by IsmaelMartinez** [comment]
Hi @benyanke  and @julian-alarcon ,
 
Just added you to this as there is always good to have an extra pair of eyes.

No rush to get this out the door.

File: app/config/README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ Here is the list of available arguments and its usage:
 | help  | show the available commands | false |
 | version | show the version number | false |
 | disableDesktopNotificationsHack | disable electron-desktop-notifications extension hack | false |
-| closeAppOnCross | Close the app when clicking the close (X) cross | false | 
+| closeAppOnCross | Close the app when clicking the close (X) cross | false |
 | partition | [BrowserWindow](https://electronjs.org/docs/api/browser-window) webpreferences partition | persist:teams-4-linux |
 | webDebug | start with the browser developer tools open  |  false |
 | url | url to open | https://teams.microsoft.com/ |
-| config | config file location | ~/.config/teams.json |
+| config | config file location | ~/.config/config.json |
```
Comment (Rondom) [inline app/config/README.md]: What is the rationale for this change? And why change the config file to a generic name like `~/.config/config.json` which does not even include the application's name? I suggest putting it into the ~/.config/teams-for-linux/ directory.

Comment (IsmaelMartinez) [reply app/config/README.md]: Thanks @Rondom , the actual location is `~/.config/teams-for-linux/config.json` wrong README.md from got knows when... I just push a change to the readme. 

Changing the name from `teams.json` to `config.json` hopefully now makes also more sense, as it is, at the end, the config that we are setting in that file.
 
Thanks again!!
